packages/whats-new/src/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { Guide } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+import { useLocale } from '@automattic/i18n-utils';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import WhatsNewPage from './whats-new-page';
+
+const WhatsNewGuide = () => {
+	const [ showGuide, setShowGuide ] = useState( true );
+	const [ whatsNewData, setWhatsNewData ] = useState( null );
+	const __ = useI18n().__;
+	const locale = useLocale();
+
+	// Load What's New list on first site load
+	useEffect( () => {
+		const proxiedWpcom = wpcom();
+		proxiedWpcom.request = proxyRequest;
+		proxiedWpcom.req
+			.get( { path: `/whats-new/list?locale=${ locale }`, apiNamespace: 'wpcom/v2' } )
+			.then( ( returnedList ) => {
+				setWhatsNewData( returnedList );
+			} );
+	}, [ locale ] );
+
+	const toggleWhatsNew = () => setShowGuide( false );
+
+	if ( ! ( whatsNewData && showGuide ) ) return null;
+
+	return (
+		<Guide
+			className="src__main"
```

Comment:
This classname doesn't feel right. Not only it doesn't correspond to the package / component it lives in, but if other packages do the same, they'll conflict with each other.

Reply:
the linter said it should be called `src__`, but you're right. I'll disable the linter for that line.